### PR TITLE
#57 pplns

### PR DESCRIPTION
**Pull request by undsoft**
Also includes stuff from this PR https://github.com/truepool/website/pull/77, so maybe review that one first.

### CODE REVIEW

File: src/app/components/farmer-details/farmer-details.component.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<table class="details-view details-view-blue table-striped">
+  <tr>
+    <th>Launcher ID</th>
+    <td>{{ farmer.launcher_id }}</td>
+  </tr>
+  <tr>
+    <th>Points</th>
+    <td>
+      {{ farmer.points_pplns | number }}
+      ({{ farmer.share_pplns / 100 | percent: '1.0-4' }})
```
[the remark this answers is not in the document]
Comment (undsoft) [reply src/app/components/farmer-details/farmer-details.component.html:10]: Fair enough :)

[the remark this answers is not in the document]
Comment (william-gr) [reply src/app/components/farmer-details/farmer-details.component.html:10]: To be clear, the API returns 0.77 for 77%
Not sure how that translates to that pipe

[the remark this answers is not in the document]
Comment (undsoft) [reply src/app/components/farmer-details/farmer-details.component.html]: Fixed

File: src/app/components/farmer-details/farmer-details.component.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<table class="details-view details-view-blue table-striped">
+  <tr>
+    <th>Launcher ID</th>
+    <td>{{ farmer.launcher_id }}</td>
+  </tr>
+  <tr>
+    <th>Points</th>
```
Comment (william-gr) [inline src/app/components/farmer-details/farmer-details.component.html:7]: Can we say "Points (24 hours)" ? Here and on the farmers table

File: src/app/app.module.ts [hunk 3]
```diff
@@ -105,5 +106,8 @@ import { poolSizeStateKey } from './stores/pool-sizes/pool-sizes.selectors';
   ],
   providers: [FileSizePipe, DecimalPipe],
   bootstrap: [AppComponent],
+  exports: [
```
Comment (aervin) [inline src/app/app.module.ts:109]: I don't think this is necessary if it's in shared.module

**Review by aervin — CHANGES_REQUESTED** [review]
LGTM apart from my note. Nice!

**Review by feydkin — APPROVED** [review]

**Pull request merged**